bot/exts/info/information.py

```diff
@@ -337,6 +338,12 @@ async def create_user_embed(self, ctx: Context, user: MemberOrUser, passed_as_me
 
         return embed
 
+    async def user_alt_count(self, user: MemberOrUser) -> tuple[str, int | str]:
+        """Get the number oif alts for the given member."""
```

Comment:
itty bitty typo `oif` -> `of`

Reply:
That's just his accent

Reply:
I have corrected my accent in https://github.com/python-discord/bot/pull/2845/commits/98eea891c25f8beb7a99685fe1daa023654d2866